mullvad: add example (#16374)
tldr-pages/tldr: pages/common/mullvad.md

- Check status of VPN tunnel:

`mullvad status`

> - Check expire date of account and obtain device name:

Reviewer: ```suggestion
- Check the account expiration date and obtain the device name:
```
